tools/buildings_to_dsm.py

```diff
@@ -17,7 +17,32 @@
 from vtk.numpy_interface import dataset_adapter as dsa
 from vtk.util import numpy_support
 from danesfield import gdal_utils
+import glob
 
+def activate_virtual_framebuffer():
+    '''
+    Activates a virtual (headless) framebuffer for rendering 3D
+    scenes via VTK.
+
+    Most critically, this function is useful when this code is being run
+    in a Dockerized notebook, or over a server without X forwarding.
+
+    * Requires the following packages:
+      * `sudo apt-get install libgl1-mesa-dev xvfb`
+    '''
+
+    import subprocess
+    import vtk
+
+    vtk.OFFSCREEN = True
```

Comment:
Is there a variable like that? What does it do?

Reply:
I can't find any reference to this variable.  Maybe we should remove this and confirm that it still works.

Reply:
I'll rerun this step with the old version.